docs/features/decorators.rst

```diff
@@ -34,71 +34,6 @@ Please note that ''stubbed'' is a decorator generator, and cannot be used as a
 "classic" Python decorator - it must be ''\@stubbed()'', not ''\@stubbed''
 (note the parenthesis).
 
-skip_if_os
```

Comment:
Not really related but cleaned them up as I saw them, I can remove this from the commit if we want to not include it in this PR.

Reply:
It's fine to do it in this PR, but why are you doing it? What has made skip_if_os obsolete?

Reply:
Good catch @lhellebr, skip_if_os was not part of the marks move, its in `decorators/host.py`, though listed here as if it were under `decorators/__init__.py`.

I'll update.